src/sider/arbeidsuforhet/NyVurdering.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement } from "react";
+import { BodyShort, Box, Button, Heading } from "@navikt/ds-react";
+import { useArbeidsuforhetVurderingQuery } from "@/data/arbeidsuforhet/arbeidsuforhetQueryHooks";
+import {
+  VurderingResponseDTO,
+  VurderingType,
+} from "@/data/arbeidsuforhet/arbeidsuforhetTypes";
+import { tilLesbarDatoMedArUtenManedNavn } from "@/utils/datoUtils";
+
+const texts = {
+  title: "Arbeidsuførhet",
+  siste: "Siste hendelse",
```

Comment:
Hadde det vært bedre med "Siste vurdering" ?

Reply:
Peter har hvertfall sagt til meg at ordlyd og sånt nødvendigvis ikke er så viktig for han i skissene, så tror vi også må være flinke på å sparre med han rundt dette når vi jobber i frontend, og ikke nødvendigvis tenke at hvis skissen sier XYZ så er det noe Peter har tenkt masse på og det er skrevet i stein 🤔 Skissen er ofc et bra utgangspunkt, men vi burde nok være flinkere på å ta en avsjekk med han når det er ting vi lurer på og ikke bare ta det for god fisk.